config/management/genesis/src/genesis.rs

```diff
@@ -42,7 +42,12 @@ impl Genesis {
             .build(
                 chain_id,
                 None,
-                OnChainConsensusConfig::V1(ConsensusConfigV1 { two_chain: true }),
+                OnChainConsensusConfig::V2(ConsensusConfigV2 {
+                    two_chain: true,
+                    decoupled_execution: true,
+                    back_pressure_limit: 10,
+                    exclude_round: 20,
+                }),
```

Comment:
So, are these just hardcoded numbers that we don't want to configure with the tool?

Reply:
those number should work, we can configure it via txns as well